r-pkg/R/es_search.R

```diff
@@ -568,15 +568,15 @@ es_search <- function(es_host
 .ValidateAndFormatHost <- function(es_host){
 
     # [1] es_host is a string
-    if (! "character" %in% class(es_host)){
-        msg <- paste0("es_host should be a string! You gave an object of type"
+    if (! is.character(es_host)){
+        msg <- paste0("es_host should be a string. You gave an object of type"
                       , paste0(class(es_host), collapse = '/'))
         log_fatal(msg)
     }
 
     # [2] es_host is length 1
     if (! length(es_host) == 1){
-        msg <- paste0("es_host should be length 1!"
+        msg <- paste0("es_host should be length 1."
```

Comment:
haha

Reply:
so important

Reply:
it felt like we were being too aggressive